transport/server.cc

```diff
@@ -618,6 +618,11 @@ future<> cql_server::connection::process_request() {
                     f.length, mem_estimate, _server._max_request_size));
         }
 
+        if (_server._config.max_concurrent_requests && _server._requests_serving > _server._config.max_concurrent_requests) {
```

Comment:
Why not default it to a very large value? That saves an extra check.

I'm not a fan of using special values to indicate meta-information (in this case, "ignore this value").

Reply:
something like `std::numeric_limits<typeof(_server._requests_serving)>::max()`

Reply:
done